### PR TITLE
Add 60XT

### PR DESCRIPTION
**Pull request by Cipulot**
## Description

Addition of 60XT PCB.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/cipulot/60xt/info.json [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        "lto": true
+    },
+    "qmk": {
+        "locking": {
+            "enabled": true,
+            "resync": true
+        }
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
```
Comment (waffle87) [inline keyboards/cipulot/60xt/info.json:16]: Generally preferred to be enabled if possible.
```suggestion
    "features": {
        "bootmagic": true,
```

File: keyboards/cipulot/60xt/info.json [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x6BC2",
+        "shared_endpoint": {
+            "keyboard": true
+        },
+        "vid": "0x6369"
+    },
+    "layouts": {
+        "LAYOUT_all": {
```
Comment (waffle87) [inline keyboards/cipulot/60xt/info.json:38]:
```suggestion
        "LAYOUT": {
```

File: keyboards/cipulot/60xt/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        KC_F5, KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                                      KC_RALT, KC_RGUI, KC_RCTL,          KC_F10
+    ),
+    [1] = LAYOUT_all(
+        _______, KC_GRV,  KC_F1,   KC_F2,   KC_F3,    KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,   KC_F11,  KC_F12,  _______, KC_DEL, _______,
+        _______, _______, _______, KC_UP,   _______,  _______, _______, _______, _______, KC_PSCR, KC_SCRL, KC_PAUSE, _______, _______, _______,         _______,
+        _______, _______, KC_LEFT, KC_DOWN, KC_RIGHT, _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______,         _______,
+        _______, _______, _______, _______, _______,  _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______,         _______,
+        _______, _______, _______, _______,                            _______,                                       _______, _______, MO(2),           _______
+    ),
+    [2] = LAYOUT_all(
```
Comment (waffle87) [inline keyboards/cipulot/60xt/keymaps/default/keymap.c:35]:
```suggestion
    [0] = LAYOUT(
        KC_F1, KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_BSPC, KC_F6,
        KC_F2, KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,          KC_F7,
        KC_F3, KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_NUHS, KC_ENT,           KC_F8,
        KC_F4, KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),            KC_F9,
        KC_F5, KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                                      KC_RALT, KC_RGUI, KC_RCTL,          KC_F10
    ),
    [1] = LAYOUT(
        _______, KC_GRV,  KC_F1,   KC_F2,   KC_F3,    KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,   KC_F11,  KC_F12,  _______, KC_DEL, _______,
        _______, _______, _______, KC_UP,   _______,  _______, _______, _______, _______, KC_PSCR, KC_SCRL, KC_PAUSE, _______, _______, _______,         _______,
        _______, _______, KC_LEFT, KC_DOWN, KC_RIGHT, _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______,         _______,
        _______, _______, _______, _______, _______,  _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______,         _______,
        _______, _______, _______, _______,                            _______,                                       _______, _______, MO(2),           _______
    ),
    [2] = LAYOUT(
```

File: keyboards/cipulot/60xt/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        KC_F5, KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                                      KC_RALT, KC_RGUI, KC_RCTL,          KC_F10
+    ),
+    [1] = LAYOUT_all(
+        _______, KC_GRV,  KC_F1,   KC_F2,   KC_F3,    KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,   KC_F11,  KC_F12,  _______, KC_DEL, _______,
+        _______, _______, _______, KC_UP,   _______,  _______, _______, _______, _______, KC_PSCR, KC_SCRL, KC_PAUSE, _______, _______, _______,         _______,
+        _______, _______, KC_LEFT, KC_DOWN, KC_RIGHT, _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______,         _______,
+        _______, _______, _______, _______, _______,  _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______,         _______,
+        _______, _______, _______, _______,                            _______,                                       _______, _______, MO(2),           _______
+    ),
+    [2] = LAYOUT_all(
```
Comment (waffle87) [inline keyboards/cipulot/60xt/keymaps/via/keymap.c:35]:
```suggestion
    [0] = LAYOUT(
        KC_F1, KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_BSPC, KC_F6,
        KC_F2, KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,          KC_F7,
        KC_F3, KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_NUHS, KC_ENT,           KC_F8,
        KC_F4, KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),            KC_F9,
        KC_F5, KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                                      KC_RALT, KC_RGUI, KC_RCTL,          KC_F10
    ),
    [1] = LAYOUT(
        _______, KC_GRV,  KC_F1,   KC_F2,   KC_F3,    KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,   KC_F11,  KC_F12,  _______, KC_DEL, _______,
        _______, _______, _______, KC_UP,   _______,  _______, _______, _______, _______, KC_PSCR, KC_SCRL, KC_PAUSE, _______, _______, _______,         _______,
        _______, _______, KC_LEFT, KC_DOWN, KC_RIGHT, _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______,         _______,
        _______, _______, _______, _______, _______,  _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______,         _______,
        _______, _______, _______, _______,                            _______,                                       _______, _______, MO(2),           _______
    ),
    [2] = LAYOUT(
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Comment by Cipulot** [comment]
Changes applied. Overlooked the `LAYOUT_all` usage instead of the `LAYOUT` in the review I did before the PR, my bad 😅

**Review by waffle87 — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**